flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/stream/StreamExecDeduplicate.java

```diff
@@ -89,6 +91,17 @@
                                     + "but there will be additional overhead."
                                     + "Default is true.");
 
+    @Experimental
+    public static final ConfigOption<Boolean>
+            TABLE_EXEC_DEDUPLICATE_MINIBATCH_ALL_CHANGELOG_ENABLED =
+                    ConfigOptions.key("table.exec.deduplicate.mini-batch.all.change-log.enabled")
```

Comment:
We can make a reverse name.
For example: `table.exec.deduplicate.mini-batch.compact-changes`, default value is false.